airflow/providers/pinecone/operators/pinecone.py

```diff
@@ -21,7 +21,7 @@
 from typing import TYPE_CHECKING, Any, Sequence
 
 from airflow.models import BaseOperator
-from airflow.providers.pinecone.hooks.pinecone import PineconeHook
+from airflow.providers.pinecone.hooks.pinecone import PineconeHook, Vector
```

Comment:
Same as comment - https://github.com/apache/airflow/pull/39688/files#r1605261158

Reply:
changed it